index.js

```diff
@@ -204,15 +223,15 @@ const image = {
 
           im(temp_path)
             .autoOrient()
-            .write(destination_path, function (err) {
+            .write(destination_path, (err) => {
               if (err) {
                 return helpers.send500(res, err);
```

Comment:
@TiddoLangerak @joostverdoorn is there anything logically to do here that im missing?